### DISCUSSION TITLE
Surround multiple characters

### DISCUSSION THREAD

**Discussion by khaitranhq** [kylechui/nvim-surround#359] [Q&A]
Hi there. I usually write Markdown in Neovim but to write bold strings (i.e `**abc**`), I have to perform twice these operations: Select text -> `S*`
Are there any better idea to reduce keystrokes? There are other cases such as `abc` -> `___abc___`.
Thank in advance!

**Comment by kylechui** [comment]
See #147 for discussion on this topic. For the time being, the way that I would personally deal with markdown stuff is:
- Define my own custom surrounds for 2, 3, etc. stars
- Define a custom surround that takes an integer as input from the user
- Dot-repeat, e.g. `ysiW*.`
